pipelines/install/04-install_proxy.yaml

```diff
@@ -12,7 +12,7 @@
       - "--certs-tar-file {{ foreman_proxy_content_certs_tar }}"
       - "--foreman-proxy-trusted-hosts {{ server_fqdn.stdout }}"
       - "--foreman-proxy-trusted-hosts {{ ansible_nodename }}"
-      - "--foreman-proxy-foreman-base-url https://{{ server_fqdn.stdout }}"
+      - "{{ ('--foreman-proxy-foreman-base-url https://%s' % server_fqdn.stdout) if pipeline_version != 'nightly' else '' }}"
```

Comment:
This definitely won't scale for future versions, I cannot really think of a great way to do this. I tried using the `version` function for comparison but that function did not exist here.  Any ideas @evgeni ?

Reply:
"that function does not exist here"?

Can you show me the invocation and the error msg?

Reply:
The trick to get something to work, is to use it the way it's intended. Or remember to plug it in.

Reply:
You're talking in riddles, but I guess that means that works now?